tasks/system/Windows.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: Set configset list
+  win_find:
+    paths: '{{ solr_configset_path }}'
+    file_type: directory
+  register: configset_output
+  when: auto_populate_configset_list
+
+- name: Populate configset list
+  set_fact:
+    configset_list: >-
+      {{ configset_output.files | map(attribute='path') | list }}
+  when: auto_populate_configset_list
+
+- name: Get config files list contained replication handler
+  win_command: >
+    findstr /r /i /n ".*requestHandler.*replication.*solr.ReplicationHandler.*"
+    solrconfig.xml
+  args:
+    chdir: '{{ solr_configset_path }}/{{ item | win_basename }}/conf'
+    removes: '{{ solr_configset_path }}/{{ item | win_basename }}/conf/solrconfig.xml'
+  register: true_configset_xml_list
+  loop: '{{ configset_list }}'
+  failed_when: False
+  changed_when: False
+
+- name: Configure solr node
+  win_lineinfile:
+    path: '{{ solr_configset_path }}/{{ item.item | win_basename }}\conf\solrconfig.xml'
+    insertafter: '.*<requestHandler name="/replication" class="solr.ReplicationHandler".*>'
+    line: '{{ solr_config_lines }}'
+    regexp:
+      '<!--\sBEGIN\ssolr\sansible\scustom\sblock(.*)
+      END\ssolr\sansible\scustom\sblock\s-->'
+  when: item.stdout_lines | length
+  loop: '{{ true_configset_xml_list.results }}'
+  notify:
+    - restart Solr Windows
+
+- name: Force all notified handlers
+  meta: flush_handlers
```

Comment:
Why do we need to flush handlers explicitly?

Reply:
cause we need working solr till the end of this role